### PR TITLE
Checkpoint README

### PR DESCRIPTION
**Pull request by Invertisment**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Framework to automate development of Algorand Assets and Smart Contracts.
 
 ## Objectives
 
-Algorand builder is an trustworthy framework for Algorand dapps (Decentralized Applications). Main goal is to make shipping Algorand applications simple, efficient, and scalable. Think about it as a truffle suite for Algorand. The framework provides following functionality:
+Algorand builder is an trustworthy framework for Algorand dapps (Decentralized Applications). Its main goal is to make shipping Algorand applications simple, efficient, and scalable. Think about it as a Truffle suite for Algorand. The framework provides following functionality:
```
Comment (robert-zaremba) [inline README.md:6]: Its -> It's

File: docs/execution-checkpoints.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Checkpoints
+
+This is a blockchain asset/contract deployment tool.
+Since blockchain data is immutable (can't be changed) there is no way to rewrite its contents and attempts will result in an error.
+To prevent this we propose a checkpont-based model for versioning the contents of the Algorand blockchain.
```
Comment (robert-zaremba) [inline docs/execution-checkpoints.md:5]: Please add more information her:

+ how user is using it, 
+ how he can add metadata using deployer
+ how he can access data in other scripts.
+ when the checkpoint is saved

**Review by robert-zaremba — APPROVED** [review]
Please add more info as in the comment.

File: README.md (hunk 1 above)
Comment (Invertisment) [reply README.md:6]: Its means that I point into the previous thing (Polish: tego).

`https://www.thefreedictionary.com/its`:

belonging to it: This is its handle.
Not to be confused with:
it's – it is: It’s cold outside.; it has: It’s been a long winter.

File: docs/execution-checkpoints.md (hunk 2 above)
Comment (Invertisment) [reply docs/execution-checkpoints.md]: Updated. Please read again (not here, in the changed files)

**Pull request merged**